### DISCUSSION TITLE
Is it possible to enable_validation on a subset of endpoints?

### DISCUSSION THREAD

**Discussion by Sheemap** [aws-powertools/powertools-lambda-python#4771] [Q&A]
Hello! We have a fairly large monolithic lambda that does not have enable_validation set.

We tried it out on a different lambda of ours, validating the schema with an OpenAPI spec, and it was a great experience!

We want to get the monolith using it as well, but Im not finding  any way to do it incrementally. Aside from making a new lambda, and its hard to find the time to migrate the whole thing at once.

Would it be possible to allow validation to be enabled on a split Router or so?

**Comment by leandrodamascena** [comment]
Hey @Sheemap! Its nice to hear that you're having a great experience with Powertools.

Unfortunately today is not possible to define validation in some endpoints once it's defined at the Resolver level. But you're not the first that asking for this support and I think we should we consider this experience.

Let me think a little bit more about that and come back here with some ideas.

Thanks

**Comment by leandrodamascena** [comment]
Convert this discussion into an issue as it's a good feature to have and make migrations/adoption easier.
